herbert_core/classes/MPIFramework/parallel_worker.m

```diff
@@ -213,22 +213,23 @@
         % of the code.
         mis.logger = @(step,n_steps,time,add_info)...
             (je.log_progress(step,n_steps,time,add_info));
-        
+
         mis.check_cancelled = @()(f_canc(je));
-        
-        
+
+
         % send first "running" log message and set-up starting time. Runs
         % asynchronously.
         n_steps = je.n_steps;
         if DO_LOGGING; log_disp_message('Logging start and checking for job cancellation before loop je.is_completed loop\n'); end
         mis.do_logging(0,n_steps);
         %%
-        
+        je = je.setup()
```

Comment:
I know you've documented setup in its place for je, but it would help to say here what you're doing so it is as documented as the rest of the workflow here